openshift-client/Dockerfile

```diff
@@ -3,15 +3,12 @@ FROM registry.access.redhat.com/ubi8/ubi:latest
 RUN cd /tmp \
   && yum -y update \
   && yum install -y wget \
-  && yum clean all && \
-  &&  rm -rf /var/cache/yum && \
+  && yum clean all \
   && wget https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz \
   && tar -xvzf oc.tar.gz \
-  && mv oc /usr/local/bin/oc-origin \
+  && mv oc /usr/local/bin/oc \
   && rm -rf oc.tar.gz
 
-ADD script.sh /usr/local/bin/oc
```

Comment:
I wonder why this has been done like this, i am not even sure if it's me who has done this.

Reply:
ah nm you are explaining it already

Reply:
It is me who have done this, This we have done as previously tekton was not supporting multiple arguments as param. Now as we can provide that, we are removing the script and the task.